Panels/Panel.js

```diff
@@ -0,0 +1,103 @@
+import {Cell, Column} from '@enact/ui/Layout';
+import {forward, handle} from '@enact/core/handle';
+import kind from '@enact/core/kind';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Slottable from '@enact/ui/Slottable';
+import SpotlightContainerDecorator, {spotlightDefaultClass} from '@enact/spotlight/SpotlightContainerDecorator';
+
+import Skinnable from '../Skinnable';
+
+import componentCss from './Panel.module.less';
+
+/**
+ * A Panel is the standard view container used inside a [Panels]{@link my-theme/Panels.Panels} view
+ * manager instance. [Panels]{@link my-theme/Panels.Panels} will typically contain several
+ * instances of these and transition between them.
+ *
+ * @class Panel
+ * @memberof my-theme/Panels
+ * @ui
+ * @public
+ */
+const PanelBase = kind({
+
+	name: 'Panel',
+
+	propTypes: /** @lends my-theme/Panels.Panel.prototype */ {
+		/**
+		 * Header for the panel.
+		 *
+		 * This can also be passed by the [Slottable]{@link ui/Slottable.Slottable} API by using a
+		 * component that has a `defaultSlot` value of `'header'`.
+		 *
+		 * @type {Node}
+		 * @public
+		 */
+		header: PropTypes.node
+	},
+
+	styles: {
+		css: componentCss,
+		className: 'panel',
+		publicClassNames: true
+	},
+
+	handlers: {
+		onScroll: handle(
+			forward('onScroll'),
+			({currentTarget}) => {
+				currentTarget.scrollTop = 0;
```

Comment:
Let's either remove this handler or add a comment on its purpose: to prevent browser-initiated scrolling of focusable elements that are within the Panel but are out of view when focused. I'd lean towards removing for the purposes of a starter theme.